src/integration/integration_api_scheduler_test.go

```diff
@@ -94,6 +122,34 @@ var _ = Describe("Integration_Api_Scheduler", func() {
 				resp.Body.Close()
 
 			})
+
+			Context("Scheduler is down", func() {
+				BeforeEach(func() {
+					stopScheduler(schedulerProcess)
+
+					policyStr = readPolicyFromFile("fakePolicyWithoutSchedule.json")
+					resp, err := attachPolicy(appId, policyStr)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(resp.StatusCode).To(Equal(http.StatusCreated))
+					resp.Body.Close()
+
+					schedulerProcess = startScheduler()
```

Comment:
Yes understand your point about having only with schedule.
The point we were trying to make by having a test without schedule was to verify the scheduler doesn't participate when there is no schedule.